src/ring_theory/localization.lean

```diff
@@ -2219,6 +2228,20 @@ lemma is_unit_map_of_injective (hg : function.injective g)
 is_unit.mk0 (g y) $ show g.to_monoid_with_zero_hom y ≠ 0,
   from map_ne_zero_of_mem_non_zero_divisors g hg y.2
 
+lemma eq_zero_of_mk'_eq_zero [algebra R K] [is_fraction_ring R K] {x : R} {y : non_zero_divisors R}
```

Comment:
Is this true as an `iff`?

Reply:
Yes, I'll add it.

Reply:
I think it might be easier to directly prove the `iff` version

Reply:
Do you mean I should erase `eq_zero_of_mk'_eq_zero` and instead prove this inside the iff statement?